src/components/RegisterFormComp.tsx

```diff
@@ -33,7 +38,12 @@ const RegisterForm: React.FC = () => {
             <IonInput
                 className="registerInput"
                 placeholder="Username"
-                onIonChange={(e: CustomEvent<InputChangeEventDetail>) => setUsername(e.detail.value ?? '')}
+                onIonChange={(e: CustomEvent<InputChangeEventDetail>) => setFirstName(e.detail.value ?? '')}
+            />
+            <IonInput
```

Comment:
This `IonInput` is a duplicate it looks like
![image](https://user-images.githubusercontent.com/14501811/107455031-58967600-6b0b-11eb-8e81-6948de406cb4.png)

Reply:
Oh that place holder should say first then last name

Reply:
Gotcha! I'll go ahead and approve the PR as soon as that's fixed!